webpack/assets/javascripts/foreman_tools.js

```diff
@@ -134,6 +134,9 @@ export function updateTable(element) {
     }
     uri.setSearch('per_page', perPage);
   }
+  if ($('#search-form').find('.autocomplete-input').val() === '') {
```

Comment:
not that i think its common, but maybe make sense to add trim, so even if there are whitespaces search gets removed.

Reply:
thanks @ohadlevy - I updated the code + added a test to check that as well. :+1: